substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfARangesSectionImpl.java

```diff
@@ -104,34 +105,31 @@ public void createContent() {
                  * Align to 2 * address size.
                  */
                 pos += DW_AR_HEADER_PAD_SIZE;
-                List<PrimaryEntry> classPrimaryEntries = classEntry.getPrimaryEntries();
-                if (classEntry.includesDeoptTarget()) {
-                    /* Deopt targets are in a higher address range so delay emit for them. */
-                    for (PrimaryEntry classPrimaryEntry : classPrimaryEntries) {
-                        if (!classPrimaryEntry.getPrimary().isDeoptTarget()) {
-                            pos += 2 * 8;
-                        }
-                    }
-                } else {
-                    pos += classPrimaryEntries.size() * 2 * 8;
-                }
+                // count 16 bytes for each normal compiled method
+                pos += classEntry.normalPrimaryEntries().reduce(0,
```

Comment:
Use
```
pos += classEntry.normalPrimaryEntries().count() * 2 * 8;
```
instead of this convoluted reduce.

Reply:
Ah, there's a `Stream::count()` method?

One day I'll give up on maintaining Byteman (target 6 bytecode) and actually start learning & using jdk7+ features. :-]

Reply:
Done.